database/scripts/main.js

```diff
@@ -89,7 +89,7 @@ function createPostElement(postId, title, text, author, authorId, authorPic) {
   var uid = firebase.auth().currentUser.uid;
 
   var html =
-      '<div class="post mdl-cell mdl-cell--12-col ' +
+      '<div class="post' + postId + ' mdl-cell mdl-cell--12-col ' +
```

Comment:
Oh.. mt intent is
class="post post-' + postId + ' mdl~
if 'post' class isn't use anywhere, 
you are right.